src/openzaak/urls.py

```diff
@@ -79,6 +80,7 @@
     path("ref/", include("notifications_api_common.urls")),
     # auth backends
     path("oidc/", include("mozilla_django_oidc.urls")),
+    path("events/", CloudEventWebhook.as_view(), name="cloudevent-webhook"),
```

Comment:
Since this is just a single endpoint and all of the other API endpoints are without a trailing slash, let's keep it consistent with that
```suggestion
    path("events", CloudEventWebhook.as_view(), name="cloudevent-webhook"),
```